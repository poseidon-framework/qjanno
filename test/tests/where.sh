qhs -H "SELECT foo,baz FROM basic.csv WHERE baz <> 'b2'"
