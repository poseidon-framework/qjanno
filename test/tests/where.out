a0 a2
c0 c2
